packages/app-store/cal-ai/components/AppSettingsInterface.tsx

```diff
@@ -0,0 +1,73 @@
+import { useSession } from "next-auth/react";
+
+export function MailLink({ subject, body }: { subject: string; body?: string }) {
+  const { data } = useSession();
+
+  return (
+    <a href={`mailto:${data?.user?.username}@cal.ai?subject=${subject}&body=${body ? body : ""}`}>
+      {subject ? subject : body}
+    </a>
+  );
+}
+export default function AppSettings() {
+  return (
+    <>
+      <div className="prose-sm prose prose-a:text-default prose-headings:text-emphasis prose-code:text-default prose-strong:text-default text-default">
+        <h2>Example questions:</h2>
```

Comment:
we are not gonna translate this app anytime soon because it only works in English best right now